app/grandchallenge/reader_studies/views.py

```diff
@@ -175,6 +175,28 @@ def form_valid(self, form):
         return response
 
 
+class ReaderStudyGroundTruth(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, DetailView
+):
+    model = ReaderStudy
+    permission_required = (
+        f"{ReaderStudy._meta.app_label}.change_{ReaderStudy._meta.model_name}"
```

Comment:
Using the meta stuff means that this isn't searchable, I think that we should just bite the bullet and use the explicit permissions.

Reply:
I've bitten all the local bullets and have updated all the permissions in this file. Given this PR is reasonably small I thought it justified.